core/src/main/java/org/elasticsearch/transport/TransportSettings.java

```diff
@@ -19,8 +19,11 @@
 package org.elasticsearch.transport;
 
 import org.elasticsearch.common.settings.Setting;
+import org.elasticsearch.common.settings.Setting.Scope;
 import org.elasticsearch.common.settings.Settings;
 
+import static org.elasticsearch.common.settings.Setting.groupSetting;
```

Comment:
Just personal preference, but I prefer the `Setting.groupSetting` version of this since it's explicit that the method is static just from looking at it in the code context, but that's probably too personal-preference-y for this :)